src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/sentencing/SentencingResourceIntTest.kt

```diff
@@ -3640,19 +3640,19 @@ class SentencingResourceIntTest : IntegrationTestBase() {
     @Nested
     inner class Validation {
       @Test
-      internal fun `404 when offender does not exist`() {
-        webTestClient.post().uri("/prisoners/AB765/sentencing")
+      internal fun `404 when case does not exist`() {
+        webTestClient.post().uri("/prisoners/${prisonerAtMoorland.nomsId}/sentencing")
           .headers(setAuthorisation(roles = listOf("ROLE_NOMIS_SENTENCING")))
```

Comment:
minor thing; it feels the noun should be "sentence" not "sentencing"

Less sure of I suppose you could either have 

POST /prisoners/{prisonerNumber}/sentence
or 
POST /prisoners/{prisonerNumber}/court-case/{courtCaseId}/sentence

Not sure if the second form is better?

Reply:
yes - I have it on my list to change to sentence